examples/kprobe/bpf/kprobe_example.c

```diff
@@ -0,0 +1,26 @@
+#include "common.h"
+#include "bpf_helpers.h"
+
+char __license[] SEC("license") = "GPL";
```

Comment:
Would you be willing to make this MIT or Dual MIT/GPL?

Reply:
sure